### PR TITLE
Adds underline styles to feedback section

### PR DESCRIPTION
**Pull request by DanielCouzens**
Please see the ticket for standard reports. https://github.com/epimorphics/standard-reports-ui/issues/114

Screenshot from 2024-08-12 14-37-33

### CODE REVIEW

**Review by bogdanadrianmarc — COMMENTED** [review]
Is there a way to get this to only underline the word and not the icon?

**Comment by jonrandahl** [comment]
> Is there a way to get this to only underline the word and not the icon?

I second @bogdanadrianmarc's comment?

File: app/assets/stylesheets/lr_common_styles/_feedback.scss [hunk 1]
```diff
@@ -2,4 +2,9 @@
   margin-top: 2em;
   padding-top: 1em;
   border-top: 2px solid $lr-green-nav;
+  p {
+    a {
+      text-decoration: underline;
+    }
```
Comment (jonrandahl) [inline app/assets/stylesheets/lr_common_styles/_feedback.scss:8]:
```css
p {
    a {
      text-decoration: underline;

      &:after {
         text-decoration: none;
      }
   }
}
```
maybe?

File: app/assets/stylesheets/lr_common_styles/_feedback.scss [hunk 2]
```diff
@@ -2,4 +2,9 @@
   margin-top: 2em;
   padding-top: 1em;
   border-top: 2px solid $lr-green-nav;
+  p {
+    a {
+      text-decoration: underline;
```
[the remark this answers is not in the document]
Comment (DanielCouzens) [reply app/assets/stylesheets/lr_common_styles/_feedback.scss:7]: Thanks Jon works a treat with a little tweak for the icon position :+1: 

Screenshot from 2024-08-13 15-24-06

**Review by jonrandahl — APPROVED** [review]
LGTM! 👍

**Pull request merged**